torch/nn/functional.py

```diff
@@ -2316,6 +2316,63 @@ def poisson_nll_loss(input, target, log_input=True, full=False, size_average=Non
     return ret
 
 
+def gaussian_nll_loss(input, target, var, eps=1e-8, full=True, reduction='mean'):
```

Comment:
1e-8 doesn't sound great here.
For single precision float, you want to use 1e-6 in general to avoid any issue. And for double precision you want something much smaller.

I think that 1e-6 and clamping (see other comment below) should be better.

Reply:
See my comment about clamping. Should I still use 1e-6 in this case?